modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/atoms/imageSelectorCoverImageAtom.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {State} from '@liferay/frontend-js-state-web';
+
+export const STR_NULL_IMAGE_FILE_ENTRY_ID = '0';
+
+const imageSelectorCoverImageAtom = State.atom<{
+	fileEntryId: string;
+	src: string;
+}>('imageSelectorCoverImage', {
+	fileEntryId: STR_NULL_IMAGE_FILE_ENTRY_ID,
+	src: '',
+});
```

Comment:
As I was saying in the description, the whole point of atoms is that they are typed, which you can see here.

As I write this, I just realized that I can actually delete the explicit type annotation (lines 20 and 21) because TS is smart enough to infer it from the default value (lines 23 and 24). I'm actually going to push one more commit on top of this right now to clean that up.